pyproject.toml

```diff
@@ -36,20 +38,25 @@ classifiers = [
 include = ["CHANGELOG.rst"]
 
 [tool.poetry.dependencies]
-python = "^3.6"
-django = "^2.2.13 || ^3.0"
+python = "^3.7"
```

Comment:
Which feature used in the package requires `^3.7`? I'm sure there is one, but just want to check, because Poetry would not install this package on a 3.6 system if we set this value. Django 3.1 supports 3.6

Reply:
I know you also removed this in the CI, so I'm sure there's a good reason, just checking :)

Reply:
See this is because of contextvars